src/test/java/code/practice/tasks/RoomServiceTestChapter2.java

```diff
@@ -0,0 +1,66 @@
+package code.practice.tasks;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Collection;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoomServiceTestChapter2 {
+
```

Comment:
Remove empty line after class declaration. Empty line is used here only when you declare method right after class name.